lib/testutils/httpmultibin.go

```diff
@@ -164,6 +164,29 @@ func getEncodedHandler(t testing.TB, compressionType httpext.CompressionType) ht
 	})
 }
 
+func getZstdBrHandler(t testing.TB) http.Handler {
+	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+		encoding := "zstd, br"
+		rw.Header().Set("Content-Type", "application/json")
+		rw.Header().Add("Content-Encoding", encoding)
+		data := jsonBody{
+			Header:      req.Header,
+			Compression: encoding,
+		}
+
+		bw := brotli.NewWriter(rw)
+		zw, _ := zstd.NewWriter(bw)
+		defer func() {
+			_ = zw.Close()
+			_ = bw.Close()
+		}()
+
+		if !assert.NoError(t, writeJSON(zw, data)) {
+			return
+		}
```

Comment:
you can use `require.NoError` instead of an if and `!assert.NoError` although in this case it doesn't really matter as this is the end of the function either way

Reply:
Will fix it, I just follow the pattern in the whole code base.

Reply:
yeah, we're trying to convert old tests with `if !assert.Something() { return }` to use `require.Something()` - not sure why they were originally written that way